claripy/strings.py

```diff
@@ -100,17 +108,21 @@ def StrSuffixOf(suffix, input_string):
     """
     return re.match(r'.*' + suffix.value + '$', input_string.value) is not None
 
+
 def StrIndexOf(input_string, substring, startIndex, bitlength):
     """
-    Return True if the concrete value of the input_string ends with suffix
-    otherwise false.
+    Return a concrete BitVector of bitlength size. Its value is the index of
+    the first occurrence of substring searching from startIndex if found
+    otherwise is -1.
```

Comment:
This is a very backwards way of saying this. The type of the result isn't as important as its meaning. Also, the english is broken. Try: `Find the first occurance at or after the start index of the given substring in the input string, or -1 if it is not found.`. The return type is specified in the `:return:` field.

Reply:
I realize you're just copying the style of what was already there, oops. You should change it, the way it was is bad!